src/content.ts

```diff
@@ -84,9 +72,24 @@ function listen(elem) {
     )
 }
 listen(window)
-document.addEventListener("readystatechange", _ =>
-    getAllDocumentFrames().forEach(f => listen(f)),
-)
+
+async function iframeMutationCallback(mutations: MutationRecord[], observer: MutationObserver) {
+    const leavegithubalone = await config.getAsync("leavegithubalone")
+
+    for (const mutation of mutations) {
+        if (mutation.type === "childList")
+            for (const node of mutation.addedNodes)
+                if ((node instanceof HTMLIFrameElement) || (node instanceof HTMLFrameElement)) {
+                    listen(node.contentDocument);
+                    observer.observe(node.contentDocument, {subtree: true, childList: true})
```

Comment:
This sets up event listeners on the inner document tree, separate from the parent. It also registers the observer in that document tree for nested iframes. It potentially has the same problem as the root observer - it needs to run before other things. From my testing, this observer is registered as soon as the `<iframe>` tag is added, so before the frame actually loads, but it may be imperfect.